src/main/java/Pesel.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+
+public class Pesel {
+
+    static public void main(String[] args) throws IllegalLengthException, WrongTypeOfDataException {
```

Comment:
to 'throws' w sygnaturze już chyba niepotrzebne, nie? ;)

Reply:
to jest potrzebne tylko gdy masz checked exceptiony (stąd też ich nazwa - muszą być albo przechwycone w try catc{}, albo przekazane wyżej w sygnaturze i ktoś wyżej jest zmuszony, aby zrobić try catch)

Reply:
Usunięte, dzięki za ten komentarz trochę mi więcej rozjaśnił :)